src/expr/src/expr/expr_unary.rs

```diff
@@ -110,33 +110,39 @@ macro_rules! gen_cast {
             { int16, float32, general_cast },
             { int16, float64, general_cast },
             { int16, decimal, general_cast },
+            { int16, varchar, general_to_string },
```

Comment:
Could you please put these `to_string` casts together? This part is currently all about casts between numeric types. 😄